subgraphs/uniswap-forks/protocols/pangolin/config/deployments/pangolin-avalanche/configurations.ts

```diff
@@ -0,0 +1,114 @@
+import { Address, BigDecimal, BigInt } from "@graphprotocol/graph-ts";
+import { Factory } from "../../../../../generated/Factory/Factory";
+import {
+  BIGINT_ZERO,
+  FeeSwitch,
+  MINIMUM_LIQUIDITY_FIVE_THOUSAND,
+  MINIMUM_LIQUIDITY_TEN_THOUSAND,
+  Network,
+  PROTOCOL_SCHEMA_VERSION,
+  RewardIntervalType,
+} from "../../../../../src/common/constants";
+import { Configurations } from "../../../../../configurations/configurations/interface";
+import {
+  PROTOCOL_SUBGRAPH_VERSION,
+  PROTOCOL_METHODOLOGY_VERSION,
+  PROTOCOL_NAME,
+  PROTOCOL_SLUG,
+} from "../../../src/common/constants";
+
+export class PangolinAvalancheConfigurations implements Configurations {
+  getNetwork(): string {
+    return Network.AVALANCHE;
+  }
+  getSchemaVersion(): string {
+    return PROTOCOL_SCHEMA_VERSION;
+  }
+  getSubgraphVersion(): string {
+    return PROTOCOL_SUBGRAPH_VERSION;
+  }
+  getMethodologyVersion(): string {
+    return PROTOCOL_METHODOLOGY_VERSION;
+  }
+  getProtocolName(): string {
+    return PROTOCOL_NAME;
+  }
+  getProtocolSlug(): string {
+    return PROTOCOL_SLUG;
+  }
+  getFactoryAddress(): string {
+    return "0xefa94DE7a4656D787667C749f7E1223D71E9FD88";
+  }
+  getFactoryContract(): Factory {
+    return Factory.bind(
+      Address.fromString("0xefa94DE7a4656D787667C749f7E1223D71E9FD88")
+    );
+  }
+  getTradeFee(): BigDecimal {
+    return BigDecimal.fromString("0.3");
+  }
+  getProtocolFeeToOn(): BigDecimal {
+    return BigDecimal.fromString("0.05");
+  }
+  getLPFeeToOn(): BigDecimal {
+    return BigDecimal.fromString("0.25");
+  }
+  getProtocolFeeToOff(): BigDecimal {
+    return BigDecimal.fromString("0");
+  }
+  getLPFeeToOff(): BigDecimal {
+    return BigDecimal.fromString("0.3");
+  }
+  getFeeOnOff(): string {
+    return FeeSwitch.ON;
```

Comment:
Did you ask in the discord about this? I was not able to verify in the docs that the protocol fee is on and what the  percentage is.

Reply:
Ah Im not sure, I think they said in the whitepapers they were actually going to start with the fees off with the option to turn it on later so perhaps I should switch this to off

Reply:
I see. Maybe double check. It looks like they have not updated with SetFeeTo when I look at the Factory contract history. So whatever it starts as should be what it is now. https://snowtrace.io/txs?a=0xefa94de7a4656d787667c749f7e1223d71e9fd88&p=2

Reply:
Yes you're right they haven't set a feeTo yet so I've turned this off